openssl/src/ssl/mod.rs

```diff
@@ -67,14 +69,25 @@ bitflags! {
         const SSL_OP_SINGLE_ECDH_USE = 0x00080000,
         const SSL_OP_SINGLE_DH_USE = 0x00100000,
         const SSL_OP_CIPHER_SERVER_PREFERENCE = 0x00400000,
-        const SSL_OP_TLS_ROLLBACK_BUG = 0x00800000,
-        const SSL_OP_NO_SSLV2 = 0x00000000,
+        const SSL_OP_TLS_ROLLBACK_BUG = 0x00800000, // 0x00000400 before 0.9.7
+        const SSL_OP_NO_SSLV2 = 0x01000000,
```

Comment:
In what versions does `SSL_OP_NO_SSLV2` exist? It doesn't seem to be present in either 0.9.8zf or 1.0.2a.